staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -52,7 +52,7 @@ type Helper struct {
 	GracePeriodSeconds  int
 	IgnoreAllDaemonSets bool
 	Timeout             time.Duration
-	DeleteLocalData     bool
```

Comment:
🤔 I wonder if removing this counts as a breaking change.

Reply:
Hey @eddiezane thanks for the review. Good question, but I am wondering which programs (other than tests) would use Helper struct?